module/nntest/module.py

```diff
@@ -811,12 +811,14 @@ def resolve_all_deps(deps, platform):
 class ActionOptions:
     def __init__(self, i):
         self.data_uoa = i.get('data_uoa','')
-        self.lib_uoa = i.get('library_uoa','')
         self.species = i.get('species','')
         self.cmd_key = i.get('cmd_key','')
         self.target = i.get('target','')
         self.user = i.get('user','')
 
+        lib_uoas = i.get('library_uoa','')
```

Comment:
@gfursin notes that usually he only allows users to provide a single UOA for something that ends with `_uoa`. Allowing to use a comma-separated list of library UOAs breaks this convention. However, if we simply change the flag to `--library`, then we can use whatever convention. 

This variable would become `library`. `self.lib_uoas` -> `self.libraries`. (I suggest `--library` because we similarly use `--dataset_file` where multiple files may be specified.)

What do you think?

Reply:
Parameter renamed